back/engines/commercial/idea_custom_fields/spec/acceptance/ideas_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Ideas' do
+  explanation 'Proposals from citizens to the city.'
+
+  let(:user) { create(:user) }
+
+  before do
+    header 'Content-Type', 'application/json'
+    @user = user
+    token = Knock::AuthToken.new(payload: @user.to_token_payload).token
+    header 'Authorization', "Bearer #{token}"
+  end
+
+  post 'web_api/v1/ideas' do
+    before do
+      IdeaStatus.create_defaults
+    end
+
+    with_options scope: :idea do
+      parameter :project_id, "The identifier of the project that hosts the idea", extra: ""
+      parameter :publication_status, "Publication status", required: true, extra: "One of #{Post::PUBLICATION_STATUSES.join(",")}"
+      parameter :title_multiloc, "Multi-locale field with the idea title", required: true, extra: "Maximum 100 characters"
+      parameter :body_multiloc, "Multi-locale field with the idea body", extra: "Required if not draft"
+      parameter :topic_ids, "Array of ids of the associated topics"
+      parameter :area_ids, "Array of ids of the associated areas"
+      parameter :custom_field_values, "a json representing custom fields"
+    end
+
+    let(:idea) { build(:idea) }
+    let(:project) { create(:continuous_project) }
+    let(:project_id) { project.id }
+    let(:publication_status) { 'published' }
+    let(:title_multiloc) { idea.title_multiloc }
+    let(:body_multiloc) { idea.body_multiloc }
+    let(:topic_ids) { create_list(:topic, 2, projects: [project]).map(&:id) }
+    let(:area_ids) { create_list(:area, 2).map(&:id) }
+
+    describe do
```

Comment:
`describe` should have an argument. I know it is optional, but for documentation purposes (run `rspec -f d` for instance) it is better to add it. It also tells the user what is being described.

Reply:
Since there is a single test with a clear enough description nested in there, I really have no idea what to write here, if you have suggestions or a better way to wrap exapmles, feel free !

Reply:
I will close this conversation, because it seems to be a common pattern.